examples/custom-resources/jwks/jwks.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: k8s.nginx.org/v1
+kind: Policy
+metadata:
+  name: jwt-policy
+spec:
+  jwt:
+    realm: MyProductAPI
+    token: $http_token
+    jwksURI: http://keycloak.default.svc.cluster.local:8080/realms/jwks-example/protocol/openid-connect/certs
```

Comment:
Lets also use < > and clarify that the user needs to use their specific IdP URL in the `jwksURI` section to remove any confusion.